source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizerTests/ADCIOSVisualizerTests.mm

```diff
@@ -186,25 +186,36 @@ - (void)parseAndRenderDFS:(NSString *)rootPath FileManager:(NSFileManager *)fMan
                                          error:nil];
 
         NSString *fileName = [pathToFile lastPathComponent];
-        ACOAdaptiveCardParseResult *cardParseResult = [ACOAdaptiveCard fromJson:payload];
-
-        if ([_setOfExpectedToFailFiles containsObject:fileName]) {
-            XCTAssertFalse(cardParseResult.isValid);
-        } else {
-            XCTAssertTrue(cardParseResult.isValid);
-            ACRRenderResult *renderResult;
-            @try {
-                renderResult = [ACRRenderer render:cardParseResult.card
-                                            config:nil
-                                   widthConstraint:300
-                                          delegate:nil];
-                XCTAssertTrue(renderResult.succeeded);
-            }
-            @catch (NSException *exception) {
-                NSLog(@"Render Failed while rendering %@\n%@", fileName, exception);
-                XCTAssertTrue(NO);
+        
+        ACOAdaptiveCardParseResult *cardParseResult = nil;
+        @try {
+            cardParseResult = [ACOAdaptiveCard fromJson:payload];
+            if ([_setOfExpectedToFailFiles containsObject:fileName]) {
+                //XCTAssertFalse(cardParseResult.isValid);
+            } else {
```

Comment:
since we don't have anything to do in the `true` case:

```suggestion
            if (![_setOfExpectedToFailFiles containsObject:fileName]) {
```

Reply:
Thanks. The comment should be uncommented. It was there trying to debug the parse failure.  It tests that said expected failure cards indeed failed.